### PR TITLE
Fix/7436 s2s validation preview tests

### LINKED ISSUE #7436: SoundnessTest for infra do not cover previews
### Feature Description
InfraValidationSoundnessTest

### Problem Description
Only checks the sharing of complete entities via S2S but not the sharing of preview objects.

### Proposed Change
Adjust tests to also include preview validation.

### LINKED ISSUE #7438: Hotfix branches are not CI checked
### Feature Description
GitHub action CI

### Problem Description
The branch/PR targets of the actions does not trigger runs for `hotfix-*` branches

### Proposed Change
Include `hotfix-*` as PR and branch target

### PR DESCRIPTION
**Pull request by JonasCir**
fix #7436
fix #7438

### CODE REVIEW

File: sormas-api/src/main/java/de/symeda/sormas/api/event/EventParticipantDto.java [hunk 1]
```diff
@@ -1,20 +1,17 @@
-/*******************************************************************************
+/*
```
Comment (vidi42) [inline sormas-api/src/main/java/de/symeda/sormas/api/event/EventParticipantDto.java:1]: Please also update the copyright year to 2016-2021

File: sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleDto.java [hunk 2]
```diff
@@ -1,20 +1,17 @@
-/*******************************************************************************
+/*
```
Comment (vidi42) [inline sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleDto.java:1]: Please also update the copyright year to 2016-2021

**Review by vidi42 — APPROVED** [review]
Approved, just please take care of the comments before merging

File: sormas-api/src/main/java/de/symeda/sormas/api/event/EventParticipantDto.java (hunk 1 above)
Comment (JonasCir) [reply sormas-api/src/main/java/de/symeda/sormas/api/event/EventParticipantDto.java:1]: done

File: sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleDto.java (hunk 2 above)
Comment (JonasCir) [reply sormas-api/src/main/java/de/symeda/sormas/api/sample/SampleDto.java:1]: done

**Pull request merged**